lib/services/android_auto_helper.dart

```diff
@@ -51,6 +51,11 @@ class AndroidAutoHelper {
   }
 
   Future<List<BaseItemDto>> getBaseItems(MediaItemId itemId) async {
+    // offline mode only supports albums and playlists for now (no offline instant mix for others yet)
+    if (FinampSettingsHelper.finampSettings.isOffline && (itemId.contentType == TabContentType.artists || itemId.contentType == TabContentType.genres)) {
+      return [];
+    }
+
```

Comment:
That is not correct any more. New download system supports albums, playlists and artists. Basically, all collections.  
Instant mixes are a different beast, but playback of artists and genres should be possible (genre playback isn't implemented yet, but that is true for online mode as well

Reply:
I managed to implement this for artists, where I just add all downloaded songs per album to queue. I'll try and see if I can also get genres to work, since it's just browsing downloaded albums I no reason why it wouldn't be possible.

Reply:
Good catch!